okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpClient.java

```diff
@@ -57,6 +60,13 @@ public RemotingOkHttpCall newCall(Request request) {
         return newCallWithMutableState(addRateLimitIdTag(request), backoffStrategyFactory.get(), MAX_NUM_RELOCATIONS);
     }
 
+    @Override
+    public Builder newBuilder() {
+        log.warn("Attempting to copy RemotingOkHttpClient. Some of the functionality like rate limiting and qos will "
+                + "not be available to the new client");
```

Comment:
could we pass a `new SafeRuntimeException("stacktrace")` to this as well, so we know where this was called and track it?